### ISSUE TITLE
Subselect with .AS('field_alias') Not Working

### ISSUE THREAD

**Issue by valeeum** [exjs/xql#5]
In your documentation, you have the following example.  But if you look at the compiled query, the "AS 'population'" part is missing.  I'm experiencing this problem on my end with a similar query.

```
Complex SQL selects are possible by combining various SQL expressions together:

var query = SELECT()
  .FIELD("name")
  .FIELD(
     SELECT(MAX(COL("pop")))
      .FROM("cities")
      .WHERE(COL("cities.state"), "=", COL("states.name"))
      .AS("population"))
  .FROM("states");
yields to:

SELECT
  "name",
  (SELECT MAX("pop") FROM "cities" WHERE "cities"."state" = "states"."name")
FROM
  "states";

```

**Comment by kobalicek** [comment]
Checking it out. As a workaround you can use `AS` closer to `"pop"` like this:

```js
return SELECT()
  .FIELD("name")
  .FIELD(
     SELECT(MAX(COL("pop")).AS("population"))
      .FROM("cities")
      .WHERE(COL("cities.state"), "=", COL("states.name"))
      .AS("population"))
  .FROM("states");
```
It's not the same query I know, will try to fix this fast.

**Comment by kobalicek** [comment]
I don't really like the fix, but until I find something better I would keep it this way. Let me know if it fixes your issue and many thanks for the report!

**Comment by valeeum** [comment]
worked like a charm.  thank you!

**Comment by kobalicek** [comment]
I released `xql@1.3.1` that has this fix in case you are using npm to resolve dependencies. Thanks!
